test/test_conversion.py

```diff
@@ -270,3 +270,13 @@ class HAVCMode(enum.Enum):
         self.assertEqual(bytes([2]), knxdclient.encode_value(HAVCMode.Standby, knxdclient.KNXDPT.ENUM8))
         self.assertEqual(bytes([2]), knxdclient.encode_value(2, knxdclient.KNXDPT.ENUM8))
         self.assertEqual(2, knxdclient.decode_value(bytes([2]), knxdclient.KNXDPT.ENUM8))
+
+    def test_dpt232_conversion(self) -> None:
+        self.assertEqual(bytes([0x10, 0xCC, 0x0A]),
+                         knxdclient.encode_value(bytes([0x10, 0xCC, 0x0A]), knxdclient.KNXDPT.COLOUR_RGB))
+        self.assertEqual(bytes([127, 00, 32]),
+                         knxdclient.encode_value(bytes([0x7F, 0x00, 0x20]), knxdclient.KNXDPT.COLOUR_RGB))
+        self.assertEqual(f'{0x801B10:X}',
```

Comment:
You don't need the f-string here; simply use the constant string literal "0x801B10". But see also my general comments about the used datatype above.

Reply:
The tests I ran kept reporting failures because [simplified example] "$FF != 255"' hence the F-string to force the conversion.

Let me re-jig it and see how I go with it.